lib/hexpm_web/controllers/api/organization_controller.ex

```diff
@@ -10,7 +10,7 @@ defmodule HexpmWeb.API.OrganizationController do
 
   plug :authorize,
        [domain: "api", resource: "read", fun: &organization_access/2]
-       when action == :show
+       when action in [:show, :audit_logs]
```

Comment:
Is this the right authorization strategy? 
An user with read access for an organization can access this org's audit logs.

Reply:
And shall we test the authorization failed case for `:audit_logs`?

Reply:
👍 It would be good to use the same test we use for `:show`.

Reply:
Added in 3c943251 Add test for GET /api/orgs/:organization/audit_logs authorization